shimmingtoolbox/__init__.py

```diff
@@ -6,3 +8,9 @@
     import importlib_metadata as metadata
 __version__ = metadata.version(__name__)
 del metadata
+
+__dir_shimmingtoolbox__ = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+__dir_testing__ = 'testing_data'
```

Comment:
I wouldn't put this here. I would put it somewhere in tests/. Maybe conftest.py?

Reply:
I am unsure what the best practice here is.